corehq/apps/hqwebapp/static/hqwebapp/spec/bootstrap3/main.js

```diff
@@ -1,16 +1,10 @@
-hqDefine("hqwebapp/spec/bootstrap3/main", [
-    "mocha/js/main",
-], function (
-    hqMocha
-) {
-    hqRequire([
-        "hqwebapp/spec/assert_properties_spec",
-        "hqwebapp/spec/bootstrap3/inactivity_spec",
-        "hqwebapp/spec/urllib_spec",
-        "hqwebapp/spec/bootstrap3/widgets_spec",
-    ], function () {
-        hqMocha.run();
-    });
+import hqMocha from "mocha/js/main";
```

Comment:
`hqRequire` no longer exists in webpack areas, and using ES6 syntax here makes the code simpler.